pkg/minikube/registry/global.go

```diff
@@ -66,10 +66,11 @@ var (
 
 // DriverState is metadata relating to a driver and status
 type DriverState struct {
-	Name     string
-	Default  bool
-	Priority Priority
-	State    State
+	Name       string
+	Default    bool
+	Preference Priority
```

Comment:
how about add a comment for what is Preference in this Struct
and also add a comment for Priority
and also a comment for Default 

so the new contributors have an easier time learning this part of the code ?

Reply:
Added comments